sdk/guides/setup-a-metagraph/building-metagraph-instances/02-building-currency-l1-instances.md

```diff
@@ -0,0 +1,192 @@
+---
+id: building-currency-l1-instances
+title: Building currency L1 Instances
+---
+
+
+Let’s create our Currency L1 😎, our L1 will need you to set up at least 3 instances. 
```

Comment:
It seems like this is describing creating 3 independent servers to host the currency L1 nodes. Don't the rest of the instructions assume that all of the layers are hosted on the same 3 servers?

Reply:
Fixed